Documentation/Setup/Config/Index.rst

```diff
@@ -1968,21 +1968,20 @@ pageTitleSeparator
    Examples
          ::
 
+         This produces a title tag with the content "website. page title"::
```

Comment:
On my end, this actually results in "website . page title" (space before and after ".").

Reply:
Right, due do not using `config.pageTitleSeparator.noTrimWrap = |||` in this example.